GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaHashset.java

```diff
@@ -0,0 +1,28 @@
+package hackkerank.day3;
+
+import java.util.HashSet;
+import java.util.Scanner;
+import java.util.Set;
+
+public class JavaHashset {
+
+    public static void main(String[] args) {
+
+        Scanner scan = new Scanner(System.in);
+        int size = scan.nextInt();
+        String[] pair_left = new String[size];
+        String[] pair_right = new String[size];
+
+        for (int i = 0; i < size; i++) {
+            pair_left[i] = scan.next();
+            pair_right[i] = scan.next();
+        }
+
+        Set<String> pairs = new HashSet<String>(size);
```

Comment:
It is recommended that the name of the collections and maps estart with singular name and them the name of the interface, example: numberList